database/table.go

```diff
@@ -21,6 +21,11 @@ type Table struct {
 	infoStore *tableInfoStore
 }
 
+// Name returns the name of the table.
```

Comment:
I suppose this is a reordering of the file? Is there any other change? If so we can keep it as is, otherwise please do the reordering in a separate PR otherwise it gets extremely difficult to review.

Reply:
It's only a reordering. I did that in a separate commit for precisely ease the review.

Reply:
Cool, thanks